envoy/TAG

```diff
@@ -1 +1 @@
-1.12.3.1
+1.13.1.1
```

Comment:
The version of `envoy` is updated both in TAG and BRANCH, but the version in .circleci/config.yaml is not updated.